treeherder/model/management/commands/cycle_data.py

```diff
@@ -272,14 +271,14 @@ def _find_ideal_chunk_size(self) -> int:
         return len(older_ids) or self._chunk_size
 
 
-class IrrelevantReposDataRemovalStrategy(RemovalStrategy):
+class IrrelevantDataRemoval(RemovalStrategy):
     """
     Removes `performance_datum` rows
     that originate from repositories other than
-    `autoland`, `mozilla-central` and `mozilla-beta`
-    that are more than 6 months old.
+    `autoland`, `mozilla-central`, `fenix`, `reference-browser` and
+    `mozilla-beta` that are more than 6 months old.
 
-    TODO: check if `fenix`, `refbrow`, `gve` repositories should be included
+    TODO: check if `refbrow`, `gve` repositories should be included
```

Comment:
`refbrow` is the same with `reference-browser`. Regarding `gve`: we can ignore it.